CadRevealRvmProvider/Tessellation/RvmTessellator.cs

```diff
@@ -40,7 +40,15 @@ TessellationLogObject tessellationLogObject
                 tessellationLogObject
             );
 
-            return new TriangleMesh(mesh, p.TreeIndex, p.Color, p.AxisAlignedBoundingBox);
+            var boundingBox = p.AxisAlignedBoundingBox;
+            if (mesh.Vertices.Length > 0)
+            {
+                // Recalculating AABB, since some instances of the transformed (p.AxisAlignedBoundingBox) AABB were incorrect
+                // Guarded agaings meshes with zero vertices, to avoid throwing an exception
```

Comment:
```suggestion
                // Guarded against meshes with zero vertices, to avoid throwing an exception. Empty meshes are filtered out later.
```